template.rb

```
@@ -0,0 +1,203 @@
+USE_LOCAL_SPREE = ENV['USE_LOCAL_SPREE'] == 'true'
+ADMIN_EMAIL = ENV['ADMIN_EMAIL'] || 'spree@example.com'
+ADMIN_PASSWORD = ENV['ADMIN_PASSWORD'] || 'spree123'
+
+def run_quietly(description, &block)
+  if VERBOSE
+    say description, :blue
+    block.call
+  else
+    # Suppress output unless there's an error
+    require 'open3'
+    stdout, stderr, status = Open3.capture3("bash -c '#{yield}'")
```

RuboCop (1.81.7)

[warning] Useless assignment to variable - `stdout`. Use `_` or `_stdout` as a variable name to indicate that it won't be used.

(Lint/UselessAssignment)